data-viewer/cypress/e2e/basic-loading/loading.cy.js

```diff
@@ -10,3 +10,17 @@ describe("Basic loading tests for test nexus file", () => {
       .should("be.visible");
   });
 });
+
+describe("Test for loading nexus file with space in name", () => {
+  beforeEach(() => {
+    // This URL DOES have a whitespace, but the underline makes it look like an underscore
```

Comment:
<img width="707" alt="image" src="https://github.com/fiaisis/plotting-service/assets/44777678/e0f8659d-8a73-4de6-9ec0-995c6fcb4bee">
What I am referring to